### ISSUE TITLE
How to set custom objective to optimize custom loss function on sklearn tuner?

### ISSUE THREAD

**Issue by abhinavsp0730** [keras-team/keras-tuner#354]
### Can we define our custom objective to optimize custom loss function on sklearn tuner?
--> when I refered to the doc string of the sklearn tuner class, It is written that we've to always set the Objective to the default value
, If so than how can we define the custom loss function to optimize?
*Note that for this `Tuner`, the `objective` for the `Oracle` should always be set
to `Objective('score', direction='max')`.* Also, `Oracle`s that exploit
Neural-Network-specific training (e.g. `Hyperband`) should not be
used with this `Tuner`.
